### DISCUSSION TITLE
New hardware platform "tiny pinball"

### DISCUSSION THREAD

**Discussion by pinguin999** [missionpinball/missionpinball#133] [General]
Hi 
We are working on our own 3D printed pinball machine. We have designed our own hardware and now we want to connect it to MPF.

Is there anywhere "better" docs on how to add hardware platforms than here: https://developer.missionpinball.org/en/dev/dev/hardware.html 🥲

If not, which hardware platform is a good and up-to-date reference to look at?

Thanks.
Kolja

**Comment by pinguin999** [comment]
I found this Video very helpful: https://www.youtube.com/watch?v=PvQVoUzL8Cc

**Comment by avanwinkle** [comment]
The most robust and up-to-date platform is FAST, but those are advanced hardware boards with lots of functionality and comm requirements. The OPP is a fairly well-vetted platform from scratch, which may also provide good reference.
